docs/source/deploying-volttron/multi-platform/multi-platform-rabbitmq-deployment.rst

```diff
@@ -65,9 +67,10 @@ To setup federation on the VOLTTRON instance, run the following command on the d
 
     vcfg --rabbitmq federation [optional path to rabbitmq_federation_config.yml]
 
-This will setup federation links to upstream servers and sets policy to make the VOLTTRON exchange *federated*.  Once a
-federation link to the upstream server is established on the downstream server, the messages published on the upstream
-server become available to the downstream server as if it were published locally.
+
+This will setup federation links to upstream servers. Once a federation link to the upstream server is established on
```

Comment:
This establishes a federation link to upstream servers.... Once the link is established between upstream and downstream servers messages flow from upstream to downstream.

Perhaps a 3 step gif would be good to show this?